Kite SDK/PSPrintSDK/NSObject+Utils.m

```diff
@@ -0,0 +1,21 @@
+//
+//  NSObject+Utils.m
+//  KitePrintSDK
+//
+//  Created by Konstadinos Karayannis on 4/10/15.
+//  Copyright (c) 2015 Deon Botha. All rights reserved.
+//
+
+#import "NSObject+Utils.h"
+
+@implementation NSObject (Utils)
+
+- (void)safePerformSelector:(SEL)selector withObject:(id)object{
+    if ([self respondsToSelector:selector]){
+        IMP imp = [self methodForSelector:selector];
```

Comment:
http://stackoverflow.com/questions/7017281/performselector-may-cause-a-leak-because-its-selector-is-unknown